example/main.js

```diff
@@ -45,6 +45,7 @@ const MAPBOX_ACCESS_TOKEN = process.env.MAPBOX_ACCESS_TOKEN ||
   'Set MAPBOX_ACCESS_TOKEN environment variable or put your token here.';
 
 const CHOROPLETHS_FILE = './example/data/sf.zip.geo.json';
+const EXTRUDED_CHOROPLETHS_FILE = './example/data/sfo.buildings.zoom13.json';
```

Comment:
I don't think putting this file in the repo is right. We'll need to find/make some other small test file instead (or load from a service), we should probably filter-branch out this file to keep the repo from getting too big.